src/server/zset_family.cc

```diff
@@ -1532,6 +1634,26 @@ bool ZSetFamily::ParseRangeByScoreParams(CmdArgList args, RangeParams* params) {
   return true;
 }
 
+void ZSetFamily::ZPopMinMax(CmdArgList args, bool reverse, ConnectionContext* cntx) {
+  string_view key = ArgS(args, 1);
+  string_view count = ArgS(args, 2);
+
+  RangeParams range_params;
+  range_params.reverse = reverse;
+  ZRangeSpec range_spec;
+  range_spec.params = range_params;
+  ScoreCount sc;
+  CHECK(SimpleAtoi(count, &sc));
```

Comment:
Lets do `(*cntx_)->SendError(kUintErr)` instead of crashing 🙂

Reply:
Thanks for your feedback, I used kInvalidIntErr instead because it also covers the case where it's not a number

Reply:
I agree its a more helpful message, but latest redis returns "must be positive" even for non-integer values
https://github.com/redis/redis/blob/b08ebff31f1e3a64ae4c537c3acc40dde7a83976/src/t_zset.c#L3962-L3964

But I guess its not that important